macros/redshift_maintenance_operation.sql

```diff
@@ -0,0 +1,27 @@
+{% macro get_vacuumable_tables() %}
+    {% set vacuumable_tables_sql %}
+        select
+            '"' || table_schema || '"."' || table_name || '"' as table_name
+        from information_schema.tables
```

Comment:
i imagine we don't want to vacuum every relation returned here -- we should probably exclude:
1. views (only where `table_type == 'BASE TABLE'`)
2. everything in the information schema (`table_schema in ('information_schema', 'pg_catalog')`)